### PR TITLE
Rlpnc 5662 master burmese transliteration

### PR DESCRIPTION
**Pull request by jedwardsBasis**

### CODE REVIEW

File: api/src/main/data/xlit_scheme-utf8.xml [hunk 1]
```diff
@@ -384,6 +384,13 @@
 <doc>ISO 11940-2 is an ISO standard for the romanization of the Thai alphabet almost identical to RTGS, published in 2007. https://en.wikipedia.org/wiki/ISO_11940-2</doc>
 </scheme>
 <scheme>
+<numeric>315</numeric>
+<name>MLCTS</name>
```
Comment (fhasanaj) [inline api/src/main/data/xlit_scheme-utf8.xml:388]: Make this lowercase, seems to be the naming convention for other names in the file

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Cumulative Release Notes for rosette-common-java-api
 
+## 37.1.1
```
Comment (iredpath) [inline RELEASE-NOTES.md:3]: We're a bit more careful (sometimes, at least) about semver here - I think this should really be 37.2.0 and not 37.1.1 in that case

**Review by jedwardsBasis — COMMENTED** [review]
got the changes mentioned here all set

File: RELEASE-NOTES.md (hunk 2 above)
Comment (iredpath) [reply RELEASE-NOTES.md]: per semver.org, updated backwards-compatible functionality deserves a minor release; point releases are only meant for bug fixes.  We definitely don't always stick to this, but that't what led to the previous release being 37.1.0 and not 37.0.5

**Review by iredpath — APPROVED** [review]

**Comment by iredpath** [comment]
This should not get released until we've sorted out what is happening with Khmer, assuming we can wait

File: api/src/main/data/xlit_scheme-utf8.xml [hunk 3]
```diff
@@ -384,6 +397,20 @@
 <doc>ISO 11940-2 is an ISO standard for the romanization of the Thai alphabet almost identical to RTGS, published in 2007. https://en.wikipedia.org/wiki/ISO_11940-2</doc>
 </scheme>
 <scheme>
+<numeric>315</numeric>
+<name>mlcts</name>
+<presentationName>MLCTS</presentationName>
+<description>Myanmar Language Commission Transcription System</description>
+<doc>The Myanmar Language Commission Transcription System (1980), also known as the MLC Transcription System (MLCTS), is a transliteration system for rendering Burmese in the Latin alphabet. https://en.wikipedia.org/wiki/MLC_Transcription_System</doc>
```
Comment (fhasanaj) [inline api/src/main/data/xlit_scheme-utf8.xml:404]: is the description necessary for the doc tag?

Comment (jedwardsBasis) [reply api/src/main/data/xlit_scheme-utf8.xml:404]: this is the same format as the Thai xlit scheme above.

**Review by fhasanaj — APPROVED** [review]

**Pull request merged**